documentation.html

```diff
@@ -392,8 +409,55 @@ <h4 class="display-inlineBlock">Toggle Button as Radio Button</h4>
               </div>
             </div>
           </section>
-          
+
         </section>
+
+
+
+        <!-- Dark button class starts here -->
+
+        <section class="instruction-block" id="dark-mode">
+          <h2>Dark-Mode Button</h2>
+          <div class="text">
+            The <code>dark-mode</code> class enables you to have a dark mode for a button by adding
+            <code>dark-mode</code> class to the button
+            or by adding <code>data-theme="dark"</code> to a container element like the div element.
+          </div>
+
+          <section class="instruction-block" id="dark-modeClass">
+            <h4 class="display-inlineBlock">Adding dark-mode as a class</h4>
+            <div class="text">
+              Adding the <code>dark-mode</code> class :
+              <br><br>
+              <button class="sbtn dashed-btn orange-btn dark-mode">Dark-Mode </button>
+            </div>
+            <div class="code-block-container">
+              <div class="script-copy">
+                <pre><code class="language-markup">&lt;button class="sbtn dashed-btn orange-btn dark-mode"&gt;Dark-Mode &lt;/button&gt;</code></pre>
+                <div class="div-copy"><button class="clipboard"></button></div>
+              </div>
+            </div>
+          </section>
+          <br><br />
```

Comment:
Remove the two brs here as well please

Reply:
okay will do

Reply:
@shahednasser done

Reply:
There's still a `<br>` here. please remove it as well